### PR TITLE
Test details for test sections

### PR DESCRIPTION
**Pull request by deads2k**
Adds test details buttons for each aggregation we have.  It allows a button from any level to say, "are the problems on this test, job, or platform different by platform" in a quick chart

### CODE REVIEW

**Comment by bparees** [comment]
1) errors are being duplicated:
image

2) I think the `no-data` cells should be gray rather than green 

3) including the "test details by platforms" button on the test details page seems inceptionish (the information is already in the table above).

4) why does one of these have buttons and the other does not?
image

File: pkg/html/releasehtml/failing_tests.go [hunk 1]
```diff
@@ -11,74 +11,74 @@ import (
...
-
-	s = s + "</table>"
+	s += rows
+	s += "</table>"
 
 	return s
 }
 
-func topFailingTestsRows(topFailingTests, prevTests []sippyprocessingv1.FailingTestResult, release string) string {
+// returns the rows to display and the names of the tests being show
```
Comment (bparees) [inline pkg/html/releasehtml/failing_tests.go:77]:
```suggestion
// returns the rows to display and the names of the tests being shown
```

**Comment by deads2k** [comment]
> 1. errors are being duplicated:

prexisting



> 3\. including the "test details by platforms" button on the test details page seems inceptionish (the information is already in the table above).

open to other names



> 4\. why does one of these have buttons and the other does not?

The one without buttons has no tests to expand.  thresholds on failures not met

**Comment by bparees** [comment]
> open to other names

i'm suggesting not having it.  It doesn't take me anywhere more useful than where i already am.

**Comment by deads2k** [comment]
> > open to other names
> 
> i'm suggesting not having it. It doesn't take me anywhere more useful than where i already am.

it does a different thing. It answers the question of "are the tests failing on this platform the same or different than the other platforms" in a way that no other view does.

File: pkg/testgridanalysis/testreportconversion/testresult.go [hunk 2]
```diff
@@ -215,7 +215,7 @@ func StandardTestResultFilter(
 		TestResultFilterFuncs{
 			FilterLowValueTestsByName,
 			FilterTooFewTestRuns(minRuns),
-			FilterSuccessfulTestResults(successThreshold),
+			//FilterSuccessfulTestResults(successThreshold),
```
Comment (bparees) [inline pkg/testgridanalysis/testreportconversion/testresult.go:218]: you can drop the TODO on line 210 w/ this change

**Pull request merged**